File: C/3170.c
#include <stdio.h>

int main() {
    int n, m, r;
    scanf("%d %d", &n, &m);
    r = m/2 - n;

    if (r > 0)
        printf("Faltam %d bolinha(s)\n", r);
    
    else
        printf("Amelia tem todas bolinhas!\n");

    return 0;
}

File: 1. Beginner/3170/3170.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) throws IOException {
		Scanner scanner = new Scanner(System.in);

		long n = scanner.nextLong();
		long m = scanner.nextLong();
		int r = (int) Math.floor(m / 2 - n);
		
        System.out.println(r > 0 ? "Faltam " + r + " bolinha(s)" : "Amelia tem todas bolinhas!");
	}
}